3p/integration.js

```diff
@@ -35,12 +35,14 @@ import {twitter} from './twitter';
 import {register, run} from '../src/3p';
 import {parseUrl} from '../src/url';
 import {assert} from '../src/asserts';
+import {taboola} from '../ads/taboola';
```

Comment:
I added a mechanism to whitelist which embed types are allowed to use `amp-embed`. After rebasing add yourself to `AMP_EMBED_ALLOWED` in this file.